server/src/main/java/org/elasticsearch/search/aggregations/PipelineAggregationBuilder.java

```diff
@@ -134,6 +131,11 @@ public void validateParentAggSequentiallyOrdered(String type, String name) {
                         + "] must have a histogram, date_histogram or auto_date_histogram as parent but doesn't have a parent"
                 );
             }
+
+            @Override
+            public void validateParentAggSequentiallyOrderedWithoutSkips(String type, String name) {
```

Comment:
This needs javadoc.  It's subtly different from the version in `ForInsideTree`, and it's not clear why they do two different things without loading the whole context.

Reply:
It always throws an exception. I'll jiggle it so it's more clear.